### DISCUSSION TITLE
GitHub Support does not respond even after 40 days ##2839135

### DISCUSSION THREAD

**Discussion by Z4NO-V2** [community/community#133258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Yes: GitHub Support does not respond even after 40 days.

40 days ago I opened a ticket in github support about a serious problem with my account, I don't know if this is the general timeframe in which tickets are resolved. 
My ticket is #2839135 if it helps

**Comment by LiteBrite82** [comment]
Hi @Z4NO-V2 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
